packages/server/graphql/public/types/helpers/getActiveTeamCountByOrgIds.ts

```diff
@@ -1,76 +1,15 @@
-import getRethink from '../../../../database/rethinkDriver'
-import {RValue} from '../../../../database/stricterR'
-import TeamMember from '../../../../database/types/TeamMember'
-import {DataLoaderWorker} from '../../../graphql'
-import isValid from '../../../isValid'
-import {InputMaybe} from '../../resolverTypes'
-import {getTeamsByOrgIds} from './getTeamsByOrgIds'
+import getTeamIdsByOrgIds from '../../../../postgres/queries/getTeamIdsByOrgIds'
+import getActiveTeamCountByTeamIds from './getActiveTeamCountByTeamIds'
 
-const THIRTY_DAYS = 1000 * 60 * 60 * 24 * 30
+// Active team is the team that completed 3 meetings with more than 1 attendee
+// and have had at least 1 meeting in the last 30 days
+// Warning: the query is very expensive
+// TODO: we need to store all calculations in the database, e.g. attendeeCount
```

Comment:
It is easier to store many of needed calculations in the database. We have very expensive queries otherwie.